pytket/docs/passes.rst

```diff
@@ -10,3 +10,9 @@ pytket.passes.script
 
 .. automodule:: pytket.passes.script
     :members: compilation_pass_from_script, compilation_pass_grammar
+
+pytket.passes.auto_rebase
```

Comment:
This shouldn't be part of this PR (I think it's already been merged?) Did you reuse the branch? Github behaves better if you create a new branch from `develop` for each PR.

Reply:
Hmm this shouldn't be here, I created a fresh branch. Not sure how I managed that. This has already been merged.